### PR TITLE
Fix/phone validation

### LINKED ISSUE #353: Make sure athlete backend validation matches frontend validation
Currently the frontend happily accepts a space n the phone number, but the backend rejects it. It's very important that any validation happening on the backend is also performed on the frontend, otherwise users will just see an error message (after their edits are lost) and will have no clue why.

- [x] Allow for client to accept spaces in the phone input
- [x] Make sure all of the spaces are removed when storing the phone number (on submit)
- [x] Phone numbers should start with `+` sign

### PR DESCRIPTION
**Pull request by ikusteu**
Fixes #353 for `CustomerForm`. If needed, the new tickets for the rest of the validations should be opened, along with adding additional cypress tests for not-yet-tested functionality.

Additionally, some edge case tests for `CustomerForm` are there as well as some new util `cy` commands.

@silviot please add the italian translation for `ValidationMessages.InvalidPhone` message

### CODE REVIEW

**Review by silviot — CHANGES_REQUESTED** [review]

File: eisbuk-admin/src/components/customers/CustomerForm/CustomerForm.tsx [hunk 1]
```diff
@@ -48,7 +48,10 @@ const CustomerValidation = Yup.object().shape({
   name: Yup.string().required(i18n.t(ValidationMessage.RequiredField)),
   surname: Yup.string().required(i18n.t(ValidationMessage.RequiredField)),
   email: Yup.string().email(i18n.t(ValidationMessage.Email)),
-  phone: Yup.string(),
+  phone: Yup.string().test({
+    test: (input) => !input || /\+[0-9]*/.test(input),
```
[the remark this answers is not in the document]
Comment (ikusteu) [reply eisbuk-admin/src/components/customers/CustomerForm/CustomerForm.tsx:52]: Rightt…test just looks for a pattern…doesn’t enforce anything…thanks, I should update that in firestore.rules as well

**Comment by ikusteu** [comment]
Fixed the requested functionlality and added test cases. I've also moved the whitespace removal to `CustomerForm`'s `onBlur` for given field (`phone`). I've also removed the commit adding whitespace manipulation to `updateCustomer` (as it is not needed anymore)

File: eisbuk-admin/cypress/integration/athlete_form_spec.ts [hunk 2]
```diff
@@ -22,23 +22,75 @@ describe("add athlete", () => {
...
-      phone: "099 1245 555",
     });
 
     cy.getAttrWith("type", "submit").click();
     // check invalid date message
     cy.contains(i18n.t(ValidationMessage.InvalidDate) as string);
-    // check invalid phone number message
+  });
+
+  it.only("doesn't allow invalid phone input format", () => {
```
Comment (silviot) [inline eisbuk-admin/cypress/integration/athlete_form_spec.ts:40]: I guess `only` was left by mistake.

**Review by silviot — APPROVED** [review]
